platform/security/sts/security-sts-server/src/main/java/ddf/security/sts/SAMLSubjectProvider.java

```diff
@@ -0,0 +1,200 @@
+/**
```

Comment:
You also need to override the equals and hashCode methods because they are still pulling from the subjectNameIDFormat instance field that you are no longer using.